swagger.json

```diff
@@ -4617,12 +4717,22 @@
                             "description":  "ID of the driver assigned to the dispatch job.",
                             "example": 444
                         },
+                        "arcbest_driver_id": {
```

Comment:
Same as above, this should be a dictionary of key=value pairs, rather than a hard coded `arcbest_driver_id` key.